scripts/core/exp-bindless-images.yml

```diff
@@ -24,6 +24,81 @@ class: $xBindlessImages
 name: "$x_exp_interop_semaphore_handle_t"
 --- #--------------------------------------------------------------------------
 type: enum
+extend: true
```

Comment:
One of my comments somehow got lost. So again:

This should include `typed_etors: True` to ensure validation checks run. In general, some/most of the fields, except for `etors` need to be repeated exactly in these extend enum types.

Reply:
Done.